hudi-common/src/main/java/org/apache/hudi/common/function/FunctionWrapper.java

```diff
@@ -70,4 +71,14 @@
       }
     };
   }
+
+  public static <V> BinaryOperator<V> throwingReduceWrapper(SerializableBiFunction<V, V, V> throwingReduceFunction) {
```

Comment:
nit: can we name this as throwableReduceWrapper ("ble" instead of "ing")

Reply:
Synced offline.  We'll still use `throwing` to be consistent with other methods in the class.